session_items.py

```diff
@@ -1,20 +1,39 @@
-from flask import session
-
-_DEFAULT_ITEMS = [
-    {'id': 1, 'status': 'Not Started', 'title': 'List saved todo items'},
-    {'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added'}
-]
-
+# from flask import session
+import requests
+import logging
+import os
+import json
+
+logging.basicConfig(level=logging.DEBUG)
+boardId = os.environ.get('boardId')
+key = os.environ.get('key')
+token = os.environ.get('token')
+
+# I understand that this is not the best wat to build urls for requests but using the built in functions 
```

Comment:
This is fine if you don't have time to retry.

But if you do want to, the syntax is e.g. `requests.get(url, params=your_params)`

Where your_params would be something like `{'key': ..., 'token': ...}`